docs/conf.py

```diff
@@ -69,7 +69,7 @@
 # The short X.Y version.
 version = u'v0.2'
 # The full version, including alpha/beta/rc tags.
-release = u'v0.2.1'
+release = u'v0.2.2'
```

Comment:
Refer to https://github.com/asyml/texar-pytorch/blob/master/docs/conf.py#L73-L75 to make this automatic

Reply:
Can you study how to maintain multiple versions in readthedocs? Like in pytorch doc: https://pytorch.org/docs/stable/index.html, users can choose the version

Reply:
@ZhitingHu Got it. Yes, I'll look into it.